src/hooks/useNotion/useNotion.js

```diff
@@ -0,0 +1,109 @@
+import { useAxios } from "../useAxios/useAxios";
+
+function useNotion() {
+  const { response, error, loading, requestData } = useAxios();
+  const NOTION_SECRET = process.env.REACT_APP_NOTION_SECRET;
+  const NOTION_DATABASE_ID = process.env.REACT_APP_NOTION_DATABASE_ID;
+
+  function generateImageBlock(url) {
+    return {
+      object: "block",
+      type: "image",
+      image: {
+        type: "external",
+        external: {
+          url: url,
+        },
+      },
+    };
+  }
+
+  function generateParagraphBlock(content) {
+    return {
+      object: "block",
+      type: "paragraph",
+      paragraph: {
+        rich_text: [
+          {
+            type: "text",
+            text: {
+              content: content,
+            },
+          },
+        ],
+      },
+    };
+  }
+
+  function generateTweetBlock(text, media) {
+    const separatedText = text.split("\n");
+    const paragraphs = separatedText.filter(p => p !== "").map((text) =>
```

Comment:
use meaningful name instead of p

Reply:
I left this by mistake and from next time I won't be repeating this again.